src/uiSelectController.js

```diff
@@ -562,14 +562,26 @@ uis.controller('uiSelectCtrl',
 
   function _handleDropDownSelection(key) {
     var processed = true;
+    var idx;
     switch (key) {
       case KEY.DOWN:
         if (!ctrl.open && ctrl.multiple) ctrl.activate(false, true); //In case its the search input in 'multiple' mode
-        else if (ctrl.activeIndex < ctrl.items.length - 1) { ctrl.activeIndex++; }
+        else if (ctrl.activeIndex < ctrl.items.length - 1) {
+          idx = ++ctrl.activeIndex;
```

Comment:
Can we just declare the `var` here? I know it leaks out anyway but it makes it clearer where its used then.